### PR TITLE
0.7.2 release notes

### PR DESCRIPTION
**Pull request by j-wags**

### CODE REVIEW

File: docs/releasehistory.rst [hunk 1]
```diff
@@ -37,9 +37,10 @@ Behavior changed
   registry when initialized with no arguments, i.e. ``ToolkitRegistry()`` and makes the
   ``register_imported_toolkit_wrappers`` argument private.
 - `PR #711 <https://github.com/openforcefield/openforcefield/pull/711>`_: The
-  setter for ``Topology.boxbox_vectors`` now infers box vectors (a 3x3 matrix) when box lengths
+  setter for :py:class:`Topology.box_vectors <openforcefield.topology.Topology>`
```
Comment (mattwthompson) [inline docs/releasehistory.rst:40]: Haha, I seem to have a quota of one typo per PR

File: docs/releasehistory.rst [hunk 2]
```diff
@@ -37,9 +37,10 @@ Behavior changed
...
+  now infers box vectors (a 3x3 matrix) when box lengths
   (a 3x1 array) are passed, assuming an orthogonal box.
-  - `PR #649 <https://github.com/openforcefield/openforcefield/pull/648>`_: Makes SMARTS
+- `PR #649 <https://github.com/openforcefield/openforcefield/pull/648>`_: Makes SMARTS
   searches stereochemistry-specific (if stereo is specified in the SMARTS) for both OpenEye
```
Comment (mattwthompson) [inline docs/releasehistory.rst:44]: Probably years too late for this to be consistent, but I noticed it's sometimes `OE` and sometimes `OpenEye`

File: docs/releasehistory.rst [hunk 3]
```diff
@@ -57,11 +58,14 @@ Behavior changed
...
   constructor, which defaults to ``DEFAULT_AROMATICITY_MODEL``.
 
-Bug Fixes
+Bugfixes
 """""""""
-- `PR #715 <https://github.com/openforcefield/openforcefield/pull/715>`_: Closes issue `Issue #475
-  <https://github.com/openforcefield/openforcefield/issues/475>`_ writing a "PDB" file using OE backend rearranges
-  the order of the atoms by pushing the hydrogens to the bottom.
+- `PR #649 <https://github.com/openforcefield/openforcefield/pull/648>`_: Prevents 2020 OE
+  toolkit from issuing a warning caused by doing stereo-specific smarts searches on certain
```
Comment (mattwthompson) [inline docs/releasehistory.rst:64]:
```suggestion
  toolkit from issuing a warning caused by doing stereo-specific SMARTS searches on certain
```

**Review by mattwthompson — APPROVED** [review]
I noticed a few tiny things, which can be ignored as desired, but nothing major 👍

**Comment by j-wags** [comment]
Ah, good calls. Thanks Matt. Looks like it's time to go ahead with the release!

File: docs/releasehistory.rst (hunk 2 above)
Comment (j-wags) [reply docs/releasehistory.rst:44]: Expanded in my last commit :-)

**Pull request merged**